htdocs/fichinter/card.php

```diff
@@ -1525,7 +1525,7 @@
 					print '<td align="right">';
 					$selectmode='select';
 					if (! empty($conf->global->INTERVENTION_ADDLINE_FREEDUREATION)) $selectmode='text';
-					$form->select_duration('duration',$objp->duree,0, $selectmode);
+					$form->select_duration('duration',$objp->duree, $selectmode);
```

Comment:
It's just an error in my repository.
It's corrected now.

2015-11-28 15:52 GMT+01:00 Laurent Destailleur <notifications@github.com>:

> In htdocs/fichinter/card.php
> <https://github.com/Dolibarr/dolibarr/pull/4070#discussion_r46080525>:
>
> > @@ -1525,7 +1525,7 @@
> >  					print '<td align="right">';
> >  					$selectmode='select';
> >  					if (! empty($conf->global->INTERVENTION_ADDLINE_FREEDUREATION)) $selectmode='text';
> > -					$form->select_duration('duration',$objp->duree,0, $selectmode);
> > +					$form->select_duration('duration',$objp->duree, $selectmode);
>
> This looks to be a regression.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Dolibarr/dolibarr/pull/4070/files#r46080525>.
>